scout/server/blueprints/cases/templates/cases/case.html

```diff
@@ -930,4 +949,327 @@ <h4 class="modal-title">Edit synopsis</h4>
+    const Y_OFFSET = 5 // make room for arrows pointing at the cytoband
+
+    const IDEOGRAM_IMAGES = ['cytoband.txt.chr1.png', 'cytoband.txt.chr2.png', 
+                             'cytoband.txt.chr3.png', 'cytoband.txt.chr4.png', 
+                             'cytoband.txt.chr5.png', 'cytoband.txt.chr6.png', 
+                             'cytoband.txt.chr7.png', 'cytoband.txt.chr8.png', 
+                             'cytoband.txt.chr9.png', 'cytoband.txt.chr10.png',
+                             'cytoband.txt.chr11.png', 'cytoband.txt.chr12.png',
+                             'cytoband.txt.chr13.png', 'cytoband.txt.chr14.png',
+                             'cytoband.txt.chr15.png', 'cytoband.txt.chr16.png',
+                             'cytoband.txt.chr17.png', 'cytoband.txt.chr18.png',
+                             'cytoband.txt.chr19.png', 'cytoband.txt.chr20.png',
+                             'cytoband.txt.chr21.png', 'cytoband.txt.chr22.png',
+                             'cytoband.txt.chrX.png', 'cytoband.txt.chrY.png'];
+
+    const ROH_IMAGES = ['roh_chr1.png', 'roh_chr2.png', 'roh_chr3.png',
+                        'roh_chr4.png', 'roh_chr5.png', 'roh_chr6.png',
+                        'roh_chr7.png', 'roh_chr8.png', 'roh_chr9.png',
+                        'roh_chr10.png', 'roh_chr11.png', 'roh_chr12.png',
+                        'roh_chr13.png', 'roh_chr14.png', 'roh_chr15.png',
+                        'roh_chr16.png', 'roh_chr17.png', 'roh_chr18.png',
+                        'roh_chr19.png', 'roh_chr20.png', 'roh_chr21.png',
+                        'roh_chr22.png', 'roh_chrX.png', 'roh_chrY.png'];
+
+    const UPD_IMAGES = ['upd_chr1.png', 'upd_chr2.png', 'upd_chr3.png',
+                        'upd_chr4.png', 'upd_chr5.png', 'upd_chr6.png', 
+                        'upd_chr7.png', 'upd_chr8.png', 'upd_chr9.png',
+                        'upd_chr10.png', 'upd_chr11.png', 'upd_chr12.png',
+                        'upd_chr13.png', 'upd_chr14.png', 'upd_chr15.png',
+                        'upd_chr16.png', 'upd_chr17.png', 'upd_chr18.png',
+                        'upd_chr19.png', 'upd_chr20.png', 'upd_chr21.png',
+                        'upd_chr22.png', 'upd_chrX.png', 'upd_chrY.png'];
+
+
+    // TODO: Handle trisomy
+    const CHROMOSOMES = ['1', '2', '3', '4', '5', '6', '7', '8', '9', '10',
+                         '11', '12', '13', '14', '15', '16', '17', '18',
+                         '19', '20', '21', '22', 'X',' Y'];
+
+
+
+
+    const NUMBER_OF_COLUMNS = 3        
+    const OFFSET_X = 60;
+    const OFFSET_Y = 30;
+
+    // skära ut kromosom
+    // "M 300 150 L 300 500 L 500 500 A 50 50 0 1 1 500 150 L 300 150"
+
+    
+
+    // Draw ROH call pictures -coverage- and UPOD pictures -color coded
+    // genome regions- onto the dashboard.
+    //
+    //
+    function draw_tracks(){
+        const CYT_HEIGHT = 50 ;
+        const CYT_WIDTH = 500 ;
+        const ROH_IMGPATH = '/public/static/roh_images/'
```

Comment:
Ok, here we need to work together. For all files that do change, the scout load case function should read roh file (and image) paths from the case config file, and pass the path here.

Reply:
Yes! I'm thinking two config parameters:

    chromograph_image_location: /some/path
    chromograph_image_destination: /some/other/path

-But some other name?

Reply:
In 4dbe309e2 I've added configurable path to image dir. Though some stuff is still static. The code expects 

1. Three separate subdirs, one for each file type
     <config_path/>
        chr_images/
         roh_images/ 
         upd_images/

2. Fixed filenames
cytoBand.txt.chr1.png
upd_chr1.png
roh_chr1.png

-I think keeping strict directory names is good in order not to mess it up. However, static file names is not.

### Solution A

.*([1-22]|X|Y)\.png

(regex might be slightly off, intension is to match every png-file which ends with 1 through 22 or X or Y)


### Solution B

Configure prefixes -still will do some sort of reqexp match..


What do you think?